### PR TITLE
Minutes from progress meeting (2023-01-19)

### PR DESCRIPTION
**Pull request by PeterKraus**
@ml-evs Feel free to add links to slides etc.

### CODE REVIEW

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+3. b) Peter Kraus:
+  - Answer: It defines the "folder structure" of files related to a single experiment, as opposed to the layout within the individual files. User adoption is quite hard, and very often you just want the raw data, without bundling it.
+ 
+3. c) Logan Ward:
+  - Suggestion: Efforts such as this one (metadata extractors) could lead to a nice recursion, where your extractor taps into other linked tools.
+  
+ --
+ 
+4. a) Steffen Brinkcmann:
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:39]:
````suggestion
4. a) Steffen Brinckmann:
````

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+- Progress in WP1, WP2, WP3 (Matthew Evans)
+- Case study: yadg (Peter Kraus)
+- Open discussion
+
+## Minutes from Discussion
+
+1. David Elbert:
+  - Suggestion: Please register for the [annual MaRDA meeting](https://www.marda-alliance.org/blog-2/marda2023/). Poster sessions for ECRs are scheduled, with prizes!
+
+--
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:18]:
```suggestion
---
```

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+
+2. a) Casper Andresen:
+  - Question: Will the transformation step in our pipeline be based on semantics, or only a pure syntax translation?
+
+2. b) Matthew Evans:
+  - Answer: LinkML allows linking to other schemas, such as Dublin Core or Schema.org. Therefore, capturing semantics should be possible. Contributions from experts are welcome!
+
+--
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:26]:
```suggestion
---
```

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+3. a) Logan Ward:
+  - Question: Interesting choice of going with a "dataschema" for yadg. What exactly is it? What are the limitations?
+
+3. b) Peter Kraus:
+  - Answer: It defines the "folder structure" of files related to a single experiment, as opposed to the layout within the individual files. User adoption is quite hard, and very often you just want the raw data, without bundling it.
+ 
+3. c) Logan Ward:
+  - Suggestion: Efforts such as this one (metadata extractors) could lead to a nice recursion, where your extractor taps into other linked tools.
+  
+ --
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:37]:
```suggestion
---
```

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+4. a) Steffen Brinckmann:
+  - Question: Is passing parameters into extractors (e.g. information about background etc.) condsidered for the schemas?
+
+4. b) Peter Kraus:
+  - Answer: From my point of view, not at the extractor level, which should not be post-processing the data values.
+
+4. c) Matthew Evans:
+  - Answer: This is a very application-dependent question. Parameters are likely to strongly depend on the extractors themselves. If we support it, it must be optional.
+  
+ --
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:48]:
```suggestion
---
```

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+
+5. a) Ken Kroenlein:
+  - Suggestion: For extractor output, the data dictionaries WG of MaRDA can provide a controlled vocabulary.
+
+5. b) Peter Kraus:
+  - Answer: I will be in touch to figure out how I can implement this in my code!
+
+ --
```
Comment (PeterKraus) [inline meetings/2-meeting-2023-01-19/minutes.md:56]:
```suggestion
---
```

File: meetings/2-meeting-2023-01-19/minutes.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  - Suggestion: Please register for the [annual MaRDA meeting](https://www.marda-alliance.org/blog-2/marda2023/). Poster sessions for ECRs are scheduled, with prizes!
+
+---
+
+2. a) Casper Andresen:
+  - Question: Will the transformation step in our pipeline be based on semantics, or only a pure syntax translation?
+
+2. b) Matthew Evans:
+  - Answer: LinkML allows linking to other schemas, such as Dublin Core or Schema.org. Therefore, capturing semantics should be possible. Contributions from experts are welcome!
+
```
Comment (ml-evs) [inline meetings/2-meeting-2023-01-19/minutes.md:25]:
```suggestion

2. Matthew Evans & Peter Kraus: 
  - Presentation of current progress and case study on [yadg](https://github.com/dgbowl/yadg) ([slides](https://docs.google.com/presentation/d/1nQD7cwEG67W5MAVjXXb-6nUklDg1AcxYs8j7WRsJjIc/edit#slide=id.g1e03a09dbfb_0_16))

2. a) Casper Andresen:
  - Question: Will the transformation step in our pipeline be based on semantics, or only a pure syntax translation?

2. b) Matthew Evans:
  - Answer: LinkML allows linking to other schemas, such as Dublin Core or Schema.org. Therefore, capturing semantics should be possible. Contributions from experts are welcome!

```

**Review by ml-evs — APPROVED** [review]
Thanks @PeterKraus, just added a suggestion with info on the slides. Feel free to merge after

**Comment by ml-evs** [comment]
Actually, no need for any more back and forth, formatting looks fine to me so I will merge now.

**Pull request merged**